packages/react-devtools-shared/src/devtools/views/Settings/ComponentsSettings.js

```diff
@@ -83,11 +86,17 @@ export default function ComponentsSettings(_: {}): React.Node {
     [setParseHookNames],
   );
 
+  const [openInEditorURLPreset, setOpenInEditorURLPreset] = useLocalStorage<
+    'vscode' | 'custom',
+  >(LOCAL_STORAGE_OPEN_IN_EDITOR_URL_PRESET, 'custom');
+
   const [openInEditorURL, setOpenInEditorURL] = useLocalStorage<string>(
     LOCAL_STORAGE_OPEN_IN_EDITOR_URL,
     getDefaultOpenInEditorURL(),
   );
 
+  const vscodeFilepath = 'vscode://file/{path}:{line}';
```

Comment:
Can you move it outside of function? Same file, but outside of the component.

Reply:
thanks for the review! let me do the same :-)